netlify.toml

```diff
@@ -55,6 +55,17 @@ package = "./netlify-plugins/cache-docusaurus-dirs-file"
   from = "https://docs.helm.sh/*"
   to = "https://helm.sh/docs/:splat"
 
+# v3 redirects
+# TODO: Uncomment after v4 release
+
+# [[redirects]]
+#   from = "/docs/topics/v2_v3_migration"
+#   to = "/docs/v3/topics/v2_v3_migration"
+
+# [[redirects]]
+#   from = "/docs/faq/changes_since_helm2"
+#   to = "/docs/v3/faq/changes_since_helm2"
+
```

Comment:
add commented out redirects for v3-only pages that will need to be redirected to the v3 url path